src/stories/Form.stories.tsx

```diff
@@ -35,13 +41,18 @@ const story = {
     },
 };
 
-function Template(args) {
+function Template(args: FormProviderProps<typeof ONYXKEYS.FORMS.TEST_FORM>) {
     // Form consumes data from Onyx, so we initialize Onyx with the necessary data here
     NetworkConnection.setOfflineStatus(false);
-    FormActions.setIsLoading(args.formID, args.formState.isLoading);
-    FormActions.setErrors(args.formID, args.formState.error);
+    FormActions.setIsLoading(args.formID, !!args.formState?.isLoading);
     FormActions.setDraftValues(args.formID, args.draftValues);
 
+    if (args.formState?.error) {
+        FormActions.setErrors(args.formID, {error: args.formState.error as MaybePhraseKey});
+    } else {
+        FormActions.clearErrors(args.formID);
+    }
```

Comment:
Before, this method `FormActions.setErrors(args.formID, args.formState.error);` was used for both setting and clearing error. But that approach wasn't aligned with our current typing.